src/lib.rs

```diff
@@ -209,6 +210,37 @@ pub trait Device: AsRawFd {
 
         Ok(driver)
     }
+
+    /// Waits for a vblank.
+    fn wait_vblank(
+        &self,
+        frame: VblankWaitFrame,
+        flags: VblankWaitFlags,
+        high_crtc: u32,
+        user_data: usize,
+    ) -> Result<VblankWaitReply, SystemError> {
+        if high_crtc > 0x1f {
+            return Err(SystemError::InvalidArgument);
+        }
+
+        let (sequence, wait_type) = match frame {
+            VblankWaitFrame::Absolute(n) => (n, drm_ffi::DRM_VBLANK_ABSOLUTE),
+            VblankWaitFrame::Relative(n) => (n, drm_ffi::DRM_VBLANK_RELATIVE),
+        };
+
+        let type_ = wait_type | (high_crtc << 1) | flags.bits();
```

Comment:
The kernel still defines `_DRM_VBLANK_HIGH_CRTC_SHIFT`, this just needs a minor tweak to the bindgen build script.